medicines/api/src/document.rs

```diff
@@ -55,31 +39,229 @@ pagination! {Documents, DocumentEdge, Document}
 
 pub async fn get_documents(
     client: &impl Search,
-    search: Option<String>,
+    search: String,
     first: Option<i32>,
     last: Option<i32>,
     before: Option<String>,
     after: Option<String>,
-) -> Documents {
-    let placeholder_names: [&str; 1000] = ["A cool document"; 1000];
-    let edges = placeholder_names
+) -> Result<Documents, anyhow::Error> {
+    // TODO: Do something with last and after.
```

Comment:
Need to do something about this. Perhaps they should be removed for now?

Reply:
From [the spec](https://relay.dev/graphql/connections.htm#sec-Arguments):

> A field that returns a Connection Type must include forward pagination arguments, backward pagination arguments, or both. These pagination arguments allow the client to slice the set of edges before it is returned.

To me, this says that we can remove `last` and `before` from this list.

Reply:
can't see much of a use case, so long as there's a `totalResults` for making the last page pagination links